modules/configuring-scale-bounds-knative.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * serverless/configuring-knative-serving-autoscaling.adoc
+
+[id="configuring-scale-bounds-knative_{context}"]
+= Configuring scale bounds Knative Serving autoscaling
+
+The `minScale` and `maxScale` annotations can be used to configure the minimum and maximum number of pods that can serve applications.
```

Comment:
Throughout this PR - if you're following the OpenShift doc guidelines, "pod" should be capitalized.

Reply:
Thank you, I had no idea! Still getting used to all the guidelines :) I've updated this across all my Serverless docs now.